src/tidy-html5.mk

```diff
@@ -17,8 +17,8 @@ define $(PKG)_BUILD
     $(MAKE) -C '$(BUILD_DIR)' -j 1 install
 
     $(if $(BUILD_STATIC),
-        cd '$(PREFIX)/$(TARGET)/lib' && mv libtidys.a libtidy.a,
-        rm -f '$(PREFIX)/$(TARGET)/lib/libtidys.a')
+        cd '$(PREFIX)/$(TARGET)/lib' && mv libtidy_static.a libtidy.a,
+        rm -f '$(PREFIX)/$(TARGET)/lib/libtidy_static.a')
```

Comment:
Does line 21 serve any purpose? Line 20 renames the file that would otherwise be deleted in line 21.

Reply:
It is a conditional - the diff context shown here is too small. Line 20 is executed in static build, line 21 in dynamic build.